apps/accounts/emails.py

```diff
@@ -137,7 +136,7 @@ def password_changed_email(user):
     }
     subject = htk_setting('HTK_ACCOUNT_EMAIL_SUBJECT_PASSWORD_CHANGED') % context
     send_email(
-        template='accounts/password_changed',
+        template=htk_setting('HTK_ACCOUNT_EMAIL_PASSWORD_CHANGED_TEMPLATE'),
```

Comment:
- Emulate what is being done on lines 104, 119
- On line 130, change the signature to take in `template=None`

```suggestion
        template=template or 'accounts/password_changed',
```

Reply:
- Additionally, modify: https://github.com/hacktoolkit/django-htk/blob/d9ee17f5f0e0ec614a08da6f12080edca774f648/apps/accounts/views.py#L511 to match what is being done on https://github.com/hacktoolkit/django-htk/blob/d9ee17f5f0e0ec614a08da6f12080edca774f648/apps/accounts/views.py#L447
- Update https://github.com/hacktoolkit/django-htk/blob/d9ee17f5f0e0ec614a08da6f12080edca774f648/apps/accounts/forms/auth.py#L65 to take in `email_template`
- Modify https://github.com/hacktoolkit/django-htk/blob/d9ee17f5f0e0ec614a08da6f12080edca774f648/apps/accounts/views.py#L477-L483 to also take in `email_template`; have it match https://github.com/hacktoolkit/django-htk/blob/d9ee17f5f0e0ec614a08da6f12080edca774f648/apps/accounts/views.py#L423-L432